src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -124,6 +124,33 @@ object SampleWdl {
     val OutputKey = "goodbye.goodbye.out"
   }
 
+  object EmptyString extends SampleWdl {
+    override def wdlSource(runtime: String = "") =
+      """
+        |task hello {
+        |  command {
```

Comment:
The code also handles empty input strings, so might as well have one of those too